tests/test_mgxs_library_distribcell/test_mgxs_library_distribcell.py

```diff
@@ -6,29 +6,39 @@
 import hashlib
 sys.path.insert(0, os.pardir)
 from testing_harness import PyAPITestHarness
+from input_set import PinCellInputSet
```

Comment:
Why are you now using the `PinCellInputSet` for this test? It seems to me that this defeats the purpose of this test of distribcell MGXS.